fabric-networking-api-v1/src/testmod/java/net/fabricmc/fabric/test/networking/play/NetworkingPlayPacketTest.java

```diff
@@ -117,6 +139,13 @@ public void onInitialize() {
 				}
 			}
 		});
+
+		ServerLoginConnectionEvents.INIT.register((handler, server) -> {
```

Comment:
Add the same test for configuration

Reply:
Maybe we want to ensure that configuration can wait until the cookies have been returned.

Reply:
Added the test. I'm not sure if we need to? A mod that wants that behaviour would probably just implement that in the rest of its configuration stuff

Reply:
Login also needs testing.

I think its imporant that login and/or configuation can be bocked until the cookie has been returned.

Reply:
What would the best way to hold the network stage be?